File: LuminarPythonPrograms/FunctionProgram/factorial.py
#create a funtion to find factoiral
num=int(input("Enter ur number"))
def fact(num):
    fact=1
    for i in range(1,num+1):
        fact*=i
    return(fact)
data=fact(num)
print(data)

# fu with arg and no return

def fact(num):
    fact=1
    for i in range(1,num+1):
        fact*=i
        print(data)

File: LuminarPythonPrograms/FrontEnd/JavaScript/FuctionalPrograms/Factorial.js
var num=Number(prompt("Enter ur number"))
var f=1
var i=2
function fact(n)
{

    while(i<=n)
    {
        f*=i
        i++
    }
    return(f)
}
var data=fact(num)
console.log("fact="+data)